src/app/clusters/bindings/BindingManager.h

```diff
@@ -53,6 +53,7 @@ struct BindingManagerInitParams
     FabricTable * mFabricTable               = nullptr;
     CASESessionManager * mCASESessionManager = nullptr;
     PersistentStorageDelegate * mStorage     = nullptr;
+    bool establishConnectionOnInit           = true;
```

Comment:
Please follow existing member naming convention.  And please document what this member does.

Reply:
@bzbarsky-apple, is this suitable documentation and name change?